drivers/segger/note_sysview.c

```diff
@@ -310,6 +366,54 @@ void sched_note_irqhandler(int irq, FAR void *handler, bool enter)
 }
 #endif
 
+#ifdef CONFIG_SCHED_INSTRUMENTATION_SYSCALL
+void sched_note_syscall_enter(int nr, int argc, ...)
+{
+  if (sysview_isenabled_syscall(nr) == 0)
```

Comment:
let's sub nr by CONFIG_SYS_RESERVED at the beginning to ensure all place use the correct value.

Reply:
Done